src/options/MapOptions.js

```diff
@@ -726,14 +783,15 @@ class MapOptions extends React.Component {
                         "planet_count": 15,
                         "specialty": 40,
                         "anomaly": 30,
-                        "wormhole": 25
+                        "wormhole": 25,
+                        "racial": 20
```

Comment:
I assume these help increase the chance of adjacency for racial systems. Are there any guarantees though?

Reply:
No adjacency guarantees, probably worth considering in the future, at the very least as an option.

Reply:
Sorry for late replies, life happened 😛 
I considered ensuring adjacency; do you think it should be an additional option or inherently part of this feature? Unsure if some would find it unreasonable to e.g. always have an asteroid field for Saar